DataBase/database_interface.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from typing import Dict, Any, Optional
+
+import firebase_admin
+from firebase_admin import credentials, db
+
+cred = credentials.Certificate("serviceAccountKey.json")
```

Comment:
it should be a const, no?

Reply:
This is the convention as suggested in firebase